brain-marks/Categories/Views/CategorySheetView.swift

```diff
@@ -105,12 +105,7 @@ struct CategorySheetView: View {
     }
     
     let columnStyle = [GridItem(), GridItem(), GridItem(), GridItem()]
-    let  categorySFSymbols = ["folder", "book", "music.note",
-                              "list.bullet", "gamecontroller", "brain.head.profile",
-                              "star", "hands.clap", "bookmark",
-                              "dollarsign.circle", "shippingbox", "laptopcomputer",
-                              "chart.bar", "gift", "suit.heart", "pin",
-                              "headphones", "moon", "crown", "person"]
+    let  categorySFSymbols = SFSymbol.initialSFSymbols + SFSymbol.NO_DECORATORS
```

Comment:
Why use `SFSymbol.initialSFSymbols + SFSymbol.NO_DECORATORS`?
This is causing a warning to be printed in the console because the ones in the `initialSFSymbols` are being shown twice

`ForEach<Array<SFSymbol>, SFSymbol, Button<ModifiedContent<ModifiedContent<ModifiedContent<ModifiedContent<Image, _EnvironmentKeyWritingModifier<Optional<Color>>>, _PaddingLayout>, _BackgroundModifier<Color>>, _ClipEffect<RoundedRectangle>>>>: the ID _folder occurs multiple times within the collection, this will give undefined results!
`

Reply:
Ah I only did it to keep your original choices at the beginning without having to go through ever SF Symbol.